torch/_torch_docs.py

```diff
@@ -2703,7 +2703,8 @@ def merge_dicts(*dicts):
 after type conversions.
 i.e. not equal to ``torch.Tensor([0.])`` or ``torch.Tensor([0])`` or
 ``torch.Tensor([False])``.
-Throws a ``RuntimeError`` if ``torch.numel() != 1``.
+Throws a ``RuntimeError`` if ``torch.numel() != 1`` (after coalescing in case
```

Comment:
Does "after coalescing" mean that `is_nonzero`  coalesces every sparse tensor?

Reply:
No but `numel()` is calculated taking the coalesced tensor / dense tensor into account, this is not an inplace operation so it will not modify the tensor

Reply:
I'm not getting the right phrase for that